client/python/modeldb/utils/ConfigUtils.py

```diff
@@ -1,68 +1,71 @@
 import sys
-import yaml
+import json
 import ConfigConstants as constants
 
 class ConfigReader(object):
     def __init__(self, filename):
         # TODO: need to deal with errors here
-        self.config =  yaml.load(file(filename, 'r'))
+        self.config =  json.load(file(filename, 'r'))
         self.validate_config()
 
     def validate_config(self):
         # TODO: what all do we expect to be here?
-        if constants.PROJECT_KEY not in self.config:
+        if 'project' not in self.config:
             print "Project must be defined in config file. Cannot proceed."
             sys.exit(-1)
-        if constants.GIT_USERNAME_KEY not in self.config:
-            print "Username must be defined in config file. Cannot proceed."
+        if not self.config.get('git').get('username'):
+            print "Git username must be defined in config file. Cannot proceed."
             sys.exit(-1)
 
     def get_project(self):
-        project = self.config[constants.PROJECT_KEY]
-        return {constants.NAME_KEY : project[constants.NAME_KEY], 
-            constants.GIT_USERNAME_KEY : self.config[constants.GIT_USERNAME_KEY], 
-            constants.DESCRIPTION_KEY : project[constants.DESCRIPTION_KEY]}
+        project = self.config['project']
+        git = self.config['git']
+        return {constants.NAME_KEY : project['name'],
+            constants.GIT_USERNAME_KEY : git['username'],
+            constants.DESCRIPTION_KEY : project.get('description', '')}
 
     def get_experiment(self, expt_name=None):
         if not expt_name:
             return None
-        
-        if constants.EXPT_KEY not in self.config:
+
+        if 'experiments' not in self.config or len(self.config['experiments']) < 1:
             print "Experiment '%s' not defined in config file. Using default." \
                 % expt_name
             return None
-        
+
         experiment = None
-        experiments = self.config[constants.EXPT_KEY]
+        experiments = self.config.get('experiments', [])
```

Comment:
let's remove the experiments section of the config. It's confusing to most users and not used a lot at the moment,